components/domains/registerV2.tsx

```diff
@@ -150,6 +151,18 @@ const RegisterV2: FunctionComponent<RegisterV2Props> = ({ domain, groups }) => {
     }
   }, [priceData, priceError, duration, domain]);
 
+  useEffect(() => {
+    if (priceError || !priceData)
+      setRenewPrice(getPriceFromDomain(1, domain).toString());
+    else {
+      // Divide the priceData by the duration to get the renewal pricevvvvvvbv---666vjjj,nb,b nnb,;,,,cx,
```

Comment:
What is this comment 😭

Reply:
Sorry Thomas gave me a broken keyboard and I didn't see I destroyed the code while fixing it 😅 I am going to update it in a few hours